modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -259,20 +259,21 @@
 	new /obj/item/clothing/suit/armor/f13/power_armor/t45d/refurb(src)
 	new /obj/item/clothing/head/helmet/f13/power_armor/t45d/refurb(src)
 
-/datum/gear/donator/kits/ripper// basically enough to set up his own clinic/lab. when off-duty.
-	name = "Rippers Belonging"
-	path = /obj/item/storage/box/large/custom_kit/ripper
+/datum/gear/donator/kits/moribund  //basically enough to set up his own clinic/lab. when off-duty. or to Rp properly as vault dwelling moribund FEV scientist man
+	name = "moribunds supplies"
+	path = /obj/item/storage/box/large/custom_kit/moribund
 	ckeywhitelist = list ("mottedesstriets")
 
-/obj/item/storage/box/large/custom_kit/ripper/PopulateContents()
+/obj/item/storage/box/large/custom_kit/moribund/PopulateContents()
+	new /obj/item/organ/tongue/lizard(src)// nightstalker tongue, cant do in proper character create to get a good looking nightstalker.
+	new /obj/item/autosurgeon(src)// ^ for tongue
 	new /obj/item/clothing/glasses/hud/health/eyepatch(src)// basic healthhud but cooler.
-	new /obj/item/melee/powered/surgripper(src)// easier than coding a skill-book giver for him to craft them
-	new /obj/item/clothing/suit/hooded/followerheavy(src)// crusader drip
-	new /obj/item/clothing/ears/headphones// hard to get for a fluff item
+	new /obj/item/clothing/suit/armored/medium/combat/duster(src)// less faction specific armor
+	new /obj/item/clothing/ears/headphones(src)// hard to get for a fluff item
 	new /obj/item/clothing/head/f13/stalkerpelt(src)// morbid. but funny.
 	new /obj/item/storage/pill_bottle/chem_tin/fixer(src)// not craftable
-	new /obj/item/book/granter/trait/midsurgery(src)// skill level for surgery of basic doctor
-	new /obj/item/book/granter/trait/chemistry(src)// chem whiz
+	new /obj/item/reagent_containers/hypospray/medipen/skillpen(src)// custom pen for skill. all doc skills.
+	new /obj/item/reagent_containers/hypospray/medipen/skillpen_na(src)// custom pen for skill. second to use to teach others medicine. dangerous
```

Comment:
This is basically an instant-kill pen; it has a 10% chance of dealing 80 brain damage every tick. 200 brain damage is lethal. Why would you get this?? You can't just use "my character is a super special powerful doctor event character" as an excuse, *ESPECIALLY* when you're getting this every round.

Reply:
I dont wanna hear poor excuses when there is a donor below me that gets revival implants and other shit round start and others get power armor. this is medical skills that are used to actually help people. it doesnt have benefit directly to a person. the brain damage variant is gonna be tweaked. it wasnt suppose to do it every tick.